src/main/scala/org/ergoplatform/nodeView/wallet/AugWalletTransaction.scala

```diff
@@ -23,7 +24,7 @@ object AugWalletTransaction {
       "address" -> e.fromProposition(box.ergoTree).toOption.map(_.toString).asJson,
       "assets" -> box.additionalTokens.toArray.toSeq.asJson,
       "creationHeight" -> box.creationHeight.asJson,
-      "additionalRegisters" -> registersEncoder(box.additionalRegisters)
+      "additionalRegisters" -> box.additionalRegisters.asInstanceOf[Map[NonMandatoryRegisterId, EvaluatedValue[SType]]].asJson
```

Comment:
Is this necessary change? Looks like the previous version was better :)

Reply:
@aslesarenko I removed cast. It uses implicitly `JsonCodecs.registersEncoder`. Do you want to call it implicitly everywhere?

Reply:
I mean the previous code is shorter and more clear. Why you changed it?

Reply:
This diff is outdated. Now it's `"additionalRegisters" -> box.additionalRegisters.asJson`.